dbms/src/main/java/org/polypheny/db/transaction/TransactionImpl.java

```diff
@@ -79,15 +80,13 @@
 
     @Getter
     private final boolean analyze;
-
+    private final AtomicLong statementCounter = new AtomicLong();
```

Comment:
@hennlo did we add this? Should we remove it?

Reply:
@cedricmendelin  no this was added by @vogti

Reply:
You're right, that is no longer necessary. I've removed it.